lib/toolchain-view.coffee

```diff
@@ -3,21 +3,25 @@ _ = require 'underscore-plus'
 {$, SelectListView} = require 'atom-space-pen-views'
 
 module.exports =
-class MultirustToolchainView extends SelectListView
+class ToolchainView extends SelectListView
   previouslyFocusedElement: null
   cmd: null
   items: null
-  multirustBinPath: null
+  toolBinPath: null
+  toolChain:"multirust"
 
   constructor: (serializedState) ->
     super
 
   initialize: ->
     super
-    @multirustBinPath = atom.config.get("tokamak.multirustBinPath")
+    @toolBinPath = atom.config.get("tokamak.toolBinPath")
+    configtoolchain = atom.config.get("tokamak.toolChain")
+    if configtoolchain
+      @toolChain = configtoolchain
     @getToolchainList(@items, @toolchainExitCallback)
     @commandSubscription = atom.commands.add 'atom-workspace',
-    'tokamak:multirust-select-toolchain': => @attach()
+    'tokamak:toolBinPath-select-toolchain': => @attach()
```

Comment:
This is shown in command palette as `toolbinpath select toolchain` can we do it more explanatory?